### PR TITLE
Fix prepare output for nfts

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Allow new nft output creationg, use minimum required storage deposit for nfts

## Links to any relevant issues

#1540 

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Not tested yet

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/account/operations/transaction/prepare_output.rs [hunk 1]
```diff
@@ -112,8 +112,7 @@ impl AccountHandle {
                     second_output_builder = second_output_builder.add_unlock_condition(
                         UnlockCondition::StorageDepositReturn(StorageDepositReturnUnlockCondition::new(
                             remainder_address,
-                            // Return minimum storage deposit + any additional required storage deposit from features
-                            // or unlock conditions
```
Comment (Thoralf-M) [inline src/account/operations/transaction/prepare_output.rs:127]: Not sure if the comment was wrong or the amount should be higher

**Review by samuel-rufi — DISMISSED** [review]

File: src/account/operations/transaction/prepare_output.rs (hunk 1 above)
Comment (Thoralf-M) [reply src/account/operations/transaction/prepare_output.rs:127]: The additional amount gets added later if needed

**Review by samuel-rufi — DISMISSED** [review]

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Changelog entry

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,7 +25,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Use `OutputWithMetadataResponse` instead of `OutputResponse`;
 - `AccountHandle::build_transaction_essence` made async;
-- Add `IssuerFeature` and `SenderFeature` to `prepare_output()` options argument;
+- Allow null nft ids in `prepare_output()` for minting;
+
+### Added
+
+- `IssuerFeature` and `SenderFeature` to `prepare_output()` options argument;
```
Comment (thibault-martinez) [inline CHANGELOG.md]: Sorry I'm being super picky here but can you put added before changed please ?

**Review by thibault-martinez — APPROVED** [review]

**Review by Tuditi — APPROVED** [review]
LGTM! :+1:

**Pull request merged**